spec/domain/content/importers/single_content_item_spec.rb

```diff
@@ -41,7 +43,7 @@ def links
       end
 
       it "imports links" do
-        expect { subject.run(content_id) }
+        expect { subject.run(partial_content_item) }
```

Comment:
I would prefer not having something called `partial_content_item`. If we are passing `content_id` and `locale`, let's be explicit in the parameters.